src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/main.yml

```diff
@@ -69,8 +69,12 @@ sylius_backend_promotion_action:
     prefix: /promotion-actions
 
 sylius_backend_tax_category:
-    resource: @SyliusWebBundle/Resources/config/routing/backend/tax_category.yml
-    prefix: /tax-categories
+    resource: |
```

Comment:
@amenophis does it work in Symfony 2.7? We need to be compatible with this version.